project-koku/koku
# Cost 2629 retire cascade delete

Fixes [COST-2629](https://issues.redhat.com/browse/COST-2629)

Changes proposed in this PR:
* Do not call cascade delete on the billing data
* This will have the side effect of no longer deleting the billing data once summary partition(s) have been dropped

Testing Instructions:
1. Checkout branch
2. Load some data
3. Force a data expiration (by date)
4. Summary data prior to the date should no longer appear in report queries despite billing records not being deleted.
